src/app/core/entity/entity-actions/entity-anonymize.service.spec.ts

```diff
@@ -0,0 +1,293 @@
+import { TestBed } from "@angular/core/testing";
+import { EntityMapperService } from "../entity-mapper/entity-mapper.service";
+import { Entity } from "../model/entity";
+import { of } from "rxjs";
+import { DatabaseEntity } from "../database-entity.decorator";
+import { DatabaseField } from "../database-field.decorator";
+import {
+  mockEntityMapper,
+  MockEntityMapperService,
+} from "../entity-mapper/mock-entity-mapper-service";
+import { comparableEntityData } from "../../../utils/expect-entity-data.spec";
+import { UpdateMetadata } from "../model/update-metadata";
+import { FileService } from "../../../features/file/file.service";
+import { CoreTestingModule } from "../../../utils/core-testing.module";
+import { DefaultDatatype } from "../default-datatype/default.datatype";
+import { FileDatatype } from "../../../features/file/file.datatype";
+import moment from "moment";
+import {
+  allEntities,
+  ENTITIES,
+  EntityWithRelations,
+  expectAllUnchangedExcept,
+} from "./entity-actions-test-utils";
+import { EntityAnonymizeService } from "./entity-anonymize.service";
+
+fdescribe("EntityAnonymizeService", () => {
```

Comment:
These tests get weird errors with a HttpResponse 404 although there shouldn't be any services making http requests ... 

I am a bit lost (and maybe becoming blind to my own code's errors). Can you help me out here, @TheSlimvReal :blush:

Reply:
The 404 came from the `MockEntityMapper` if the `.get()` call fails. It happened because a entity was never saved and the testing function tries to load it.

I also fixed some other tests that did not correctly check for the `inactive` and `anonymized` flag but for the cascading ones it got a bit difficult and I was not sure whether you had a different idea in mind to get these running.

Reply:
Thanks! :heart: